gatsby-theme-landing-page/src/utils/index.js

```diff
@@ -0,0 +1,41 @@
+import * as sanitizeHTML from "sanitize-html";
+
+// This set of HTML tags are derived from the resulting output of
+// all the possible rich text options in Contentful's rich text editor.
+// The intent is to create a 1:1 mapping of what content editors can do
+// with that editor and what markup our components will support
+const defaultAllowedTags = [
+  "a",
+  "p",
+  "br",
+  "em",
+  "strong",
+  "ul",
+  "ol",
+  "li",
+  "h1",
+  "h2",
+  "h3",
+  "del",
+  "table",
+  "thead",
+  "tbody",
+  "td",
+  "th",
+  "tr",
+  "blockquote",
+  "hr",
+  "code",
```

Comment:
Do we want to support `<iframe>` here as well? I'd guess that some content editors might be expecting to be able to paste embed code into the long text field

Reply:
adding